services/slam/builtin/builtin.go

```diff
@@ -256,9 +256,10 @@ func runtimeServiceValidation(
 type AttrConfig struct {
 	Sensors             []string          `json:"sensors"`
 	ConfigParams        map[string]string `json:"config_params"`
+	DataDirectory       string            `json:"data_dir"`
+	UseLiveData         *bool             `json:"use_live_data"`
```

Comment:
currently using *bool to allow a validation check that use_live_data was included in the config